### DISCUSSION TITLE
Flybywire not working after SU5

### DISCUSSION THREAD

**Discussion by shanenicholson88** [flybywiresim/aircraft#5476] [Help]
Good morning team,

Anyone else having issues running the flybywire 320 after SU5? I cant even load it in the sim now.. Bizarre.

**Comment by ContactRon5** [comment]
Yes, several issues post SU5. Sometimes msfs will crash and I have to reload or when flying the process will hang. If the latter then weird solution - start Task Manager, highlight MSFS process then go back to MSFS and it resumes. Something not right with their upgrade.
As far as FBW goes, apart from the already documented issues, only the development installation works fully for me. BUT, and this is really weird, I have a thrustmaster X  joystick where the thrust slider works in FBW stable and test models, but since SU5, it's reversed in Development ?  So engines idle when slider is at max thrust and they're at max thrust when slider is at idle.  I guess it'll take a few weeks until things settle or patches start to appear. Must say tho that when it does work, I think the whole experience is improved under SU5. Bit weird flying at FL200 last night and the outside air temp is +38C but hey ho, it'll be rectified thru time.

**Reply by shanenicholson88** [reply]
Hey,

Thanks for taking the time to reply. I'm flying in VR too and since the update it's been a little odd 
Il take your advice but thankfully today we're seeing a hot fix around 8pm BST.

**Comment by tracernz** [comment]
Sim hotfix that is supposed to fix many of the issues tomorrow. Standby.

**Reply by MrAstroApe** [reply]
I have the same issue. Game will CTD during loading or a few seconds after start the game. Another issue is the buttons of the FMGS flashing. Hope all these issues can be fixed soon.

**Comment by mickmcp** [comment]
Same for me, development wont even fully load before the sim crashes, stable version loads but all the lights are flickering.

I did a clean install of flight sim late last night and have got version 1.18.14.0 which should include the hot fix, nothing in the community folder apart from FBW airbus. Hot fix doesn't seem to do anything for me at all with any aircraft.

**Reply by shanenicholson88** [reply]
Hey, if it helps I did a full clean install and used the flybywire tool to install the latest DEV version and it works now..

**Reply by yuriks59** [reply]
It works, but it's kind of crooked. Why did you do that when you hover the mouse over the control, the buttons and toggle switches are highlighted and switching to the desired position is difficult. It is also unclear with the headlights, the position of the toggle switches is mixed up, the front and left side lights on the landing gear do not light at all.

**Reply by mickmcp** [reply]
Stable version wont even work for me now, when it did work a couple of the light switches wouldn't work either. 

All the effort that has been put into this for free and microsoft have ruined everything chasing the money again, SU5 is the worse bit of software  I have ever seen.

**Reply by ContactRon5** [reply]
mickmcp - see my post below.  I'm gussing FBW will not update the stable version until their model is err...  stable.   Use the DEV model as per their instructions.

**Comment by ContactRon5** [comment]
yuriks59 - Sounds very much like you are not using the development model. On the FBW twitter page, they say the dev model has been modified for use with su5, they don't mention any other model. If you use the stable or experimental models, then as you say, some buttons don't work, and backlit buttons on the MCDU flicker etc.  The Development model addresses these issues. Seems to be working well for me, especially after applying the MSFS hotfix.

I must say I'm impressed. The MSFS process no longer halts after the hotfix was applied and apart from more efficient graphics, su5 has meant ground handling is much smoother. I've flown EGGP - EGPH - EGLL since the hotfix using the FBW Dev model and have to say AP even flew all the waypoints along the approach in to EGLL set in the FMS without fault. That's a first for me!

**Reply by yuriks59** [reply]
I use the 320 FBDev model and constantly update it through the FlyByWire Installer, but after the SU 5 updates, the normal operation of the sim began to collapse. I tried to put other options the same. However, the route flew normally on the IFR and with a landing on the ILS. But it is very difficult to highlight the backlit buttons and control switches when you hover the mouse cursor.

**Reply by ContactRon5** [reply]
Just a thought. When you say 'hover' the mouse cursor,  under Options / General Options / Accessibility / User Interface, do you have 'Cockpit Interation System' set to LEGACY? If not, then try that.

**Reply by yuriks59** [reply]
I have the "Commit"mode in the settings you specified. Speaking about the new functions after the general update of the sim, I meant that it used to be realistic, I hover the mouse cursor over the toggle switch, an arrow appears and everything is conveniently controlled, but now the toggle switch or button turns blue when yellow is highlighted and it is difficult to adjust, for example, the speed or flight level. It became unreal as in the simulator, but as in the flying game. We must still strive for realism.

**Reply by yuriks59** [reply]
thanks for the hint, I set the "classic" mode in the settings and everything worked as it was before, the toggle switches and levers are now not highlighted when you hover the mouse cursor over them, and all the necessary modes are installed without problems.

**Comment by elvedyn** [comment]
Just for info.. after update su5 and hotfix no crashes.. i use dev mod (
but today i use LIRF and LYBE airport from fligtsim.to and crash ) after i
remove from comunity it works :) sry for my bad english :)

Den sön 1 aug. 2021 17:37ContactRon5 ***@***.***> skrev:

**Reply by yuriks59** [reply]
Yes, after installing the latest developer update, the functions of the external light switch buttons were restored, the headlights began to work as it should. Most of the mods in the community had to be removed, since the steam does not load or throws it on the desktop.

**Comment by yuriks59** [comment]
after the update, another problem appeared, after dialing the level set on the altimeter and setting the barometer value to the standard, the ATC dispatcher informs about the elevation increase by hundreds of feet from 600 to 900.

**Reply by mickmcp** [reply]
Still the same for me, development aircraft causes a CTD before even loading in, clean install including the hotfix and nothing in the community folder. I flew all over with it many many times before the SU5 update, nothing has changed on my PC either.

**Comment by mickmcp** [comment]
I can say the development version seems to be working ok for me now since the second hotfix, no flashing buttons either.
